main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section*{Methods}
+
```

Comment:
Might be nice to add something up here about the github repo. E.g.:

"All scripts and Jupyter notebooks used to generate the analyses, figures and tables for this paper are available from the GitHub repository at https://github.com/malariagen/agam-vgsc-report".